oauth2/oauth2-jose/src/main/java/org/springframework/security/oauth2/jwt/NimbusJwtDecoder.java

```diff
@@ -312,6 +328,8 @@ public JwkSetUriJwtDecoderBuilder cache(Cache cache) {
 			// Spring Security validates the claim set independent from Nimbus
 			jwtProcessor.setJWTClaimsSetVerifier((claims, context) -> { });
 
+			jwtProcessorCustomizer.accept(jwtProcessor);
```

Comment:
For consistency with the rest of the code here, will you please prefix member variable references with `this.`?